apps/api/lib/api/web/controllers/friendship_controller.ex

```diff
@@ -0,0 +1,17 @@
+defmodule API.Web.FriendshipController do
+  use API.Web, :controller
+
+  action_fallback API.Web.FallbackController
+
+  def create(conn, %{"requestee_id" => requestee_id}) do
```

Comment:
@camirmas do you think we should pass in the user_id from the client and save ourselves the user lookup, or just always rely on the token for user information?